src/theory/quantifiers/ce_guided_single_inv.h

```diff
@@ -181,6 +179,15 @@ class CegConjectureSingleInv {
   std::map< Node, Node > d_templ;
   // the template argument for each function to synthesize (occurs in exactly one position of its template)
   std::map< Node, Node > d_templ_arg;
+  
+ public:
```

Comment:
Again, merge public blocks and add comments to separate methods. Move public section on top.